backend/keepalived/keepalived.go

```diff
@@ -329,6 +330,9 @@ func (k *Keepalived) dead(key string, prev liveness.State, leader bool) {
 		lager.Error(err)
 		return
 	}
+	event := createKeepaliveEvent(e)
```

Comment:
I believe this might cause a compile time error (see L382).

Reply:
But, those are outside of the changes I made as part of this commit.

Reply:
Yeah, I totally understand that. I believe we had a long lived branch for some of those keepalive improvements (that's on us). We just need to make sure master will compile before we merge. We can take care of this for you if you'd like 😄

Reply:
Can you, please?